src/nnet3/nnet-component-itf.cc

```diff
@@ -59,6 +59,8 @@ ComponentPrecomputedIndexes* ComponentPrecomputedIndexes::NewComponentPrecompute
     ans = new StatisticsExtractionComponentPrecomputedIndexes();
   } else if (cpi_type == "StatisticsPoolingComponentPrecomputedIndexes") {
     ans = new StatisticsPoolingComponentPrecomputedIndexes();
+  } else if (cpi_type == "SelfAttentionComponentPrecomputedIndexes") {
```

Comment:
@tomkocse, this seems to be a bug fix, can you please make a separate PR for it?

Reply:
@danpovey I don't see the bug here in the original code, can you point that out ?

Reply:
oh sorry, I didn't realize that SelfAttentionComponent was new.